File: Iniciante/1019 - Conversão de Tempo/Python/1019.py
def main():
    duration_in_seconds = int(input())
    SECONDS_PER_HOUR = 3600
    SECONDS_PER_MINUTE = 60
    hours, remainder_seconds = divmod(duration_in_seconds, SECONDS_PER_HOUR)
    minutes, seconds = divmod(remainder_seconds, SECONDS_PER_MINUTE)
    print(f"{hours}:{minutes}:{seconds}")


if __name__ == "__main__":
    main()

File: Iniciante/1019 - Conversão de Tempo/JavaScript/1019.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const durationInSeconds = parseInt(input);
  const SECONDS_PER_HOUR = 3600;
  const SECONDS_PER_MINUTE = 60;
  const hours = Math.trunc(durationInSeconds / SECONDS_PER_HOUR);
  const remainingSeconds = durationInSeconds % SECONDS_PER_HOUR;
  const minutes = Math.trunc(remainingSeconds / SECONDS_PER_MINUTE);
  const seconds = Math.trunc(remainingSeconds % SECONDS_PER_MINUTE);
  console.log(`${hours}:${minutes}:${seconds}`);
}

main();
